maintenance_evolution.pt.Rmd

```diff
@@ -96,33 +96,33 @@ foo <- function(x) {
 bar <- function(x) x + 1
 ```
 
-After users have used the package version for a while (with both `foo` and `bar`), in the next version you can remove the old function name (`foo`), and only have `bar`.
+Depois que os usuários tiverem usado a versão do pacote por algum tempo (com ambos os `foo` e `bar`), na próxima versão você poderá remover o nome da função antiga (`foo`), e você terá apenas `bar`.
 
 ```r
 #' bar - add 1 to an input
 #' @export
 bar <- function(x) x + 1
 ```
 
-## Functions: deprecate \& defunct {#functions-deprecate-defunct}
+## Funções: depreciadas e extintas {#functions-deprecate-defunct}
```

Comment:
I chose defunct as "removed" in this line after reading [.Defunct](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/Defunct) documentation. In some lines below a use defunct wrapped in quotes to reference the function name itself. Not sure how it is normally translated in R context though.

```suggestion
## Funções: descontinuadas e removidas {#functions-deprecate-defunct}
```